src/main/scala/my_gamora.scala

```diff
@@ -0,0 +1,12 @@
+object my_gamora extends App{
```

Comment:
object and class names should have first letter of each work capitalised, try to avoid using underscores in the names